tests/queries_/tests.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import Value
+from django.test import TestCase
+
+from django_mongodb_backend.test import MongoTestCaseMixin
+
+from .models import Author
+
+
+class SaveUpdateDollarPrefixTests(MongoTestCaseMixin, TestCase):
+    def test_insert_dollar_prefix(self):
+        """Ensure $-prefixed values are correctly saved on insert."""
+        obj = Author.objects.create(name="$foobar")
+        refreshed = Author.objects.get(pk=obj.pk)
+        self.assertEqual(refreshed.name, "$foobar")
+
+    def test_update_dollar_prefix(self):
+        """Ensure $-prefixed values are correctly saved on update."""
+        obj = Author.objects.create(name="foobar")
+        with self.assertNumQueries(1) as ctx:
+            obj.name = "$updated"
+            obj.save()
+        refreshed = Author.objects.get(pk=obj.pk)
+        self.assertEqual(refreshed.name, "$updated")
+        self.assertUpdateQuery(
+            ctx.captured_queries[0]["sql"],
+            "queries__author",
+            {"_id": obj.id},
+            [{"$set": {"name": {"$literal": "$updated"}}}],
+        )
```

Comment:
Per https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style, "In test docstrings, state the expected behavior that each test demonstrates. Don’t include preambles such as “Tests that” or “Ensures that”."